spec/pagseguro/authorization_request/request_serializer_spec.rb

```diff
@@ -1,23 +1,285 @@
-require "spec_helper"
+require 'spec_helper'
+
+shared_examples 'when address is assigned' do
+  it 'can be a postal code' do
+    address[:postal_code] = '01452002'
+
+    postal_code_parsed = node.at_css('> postalCode').text
+    expect(postal_code_parsed).to eq '01452002'
+  end
+
+  it 'can be a street' do
+    address[:street] = 'Av. Brig. Faria Lima'
+
+    street_parsed = node.at_css('> street').text
+    expect(street_parsed).to eq 'Av. Brig. Faria Lima'
+  end
+
+  it 'can be a number' do
+    address[:number] = '1384'
+
+    number_parsed = node.at_css('> number').text
+    expect(number_parsed).to eq '1384'
+  end
+
+  it 'can be a complement' do
+    address[:complement] = '5o andar'
+
+    complement_parsed = node.at_css('> complement').text
+    expect(complement_parsed).to eq '5o andar'
+  end
+
+  it 'can be a district' do
+    address[:district] = 'Jardim Paulistano'
+
+    district_parsed = node.at_css('> district').text
+    expect(district_parsed).to eq 'Jardim Paulistano'
+  end
+
+  it 'can be a city' do
+    address[:city] = 'Sao Paulo'
+
+    city_parsed = node.at_css('> city').text
+    expect(city_parsed).to eq 'Sao Paulo'
+  end
+
+  it 'can be a state' do
+    address[:state] = 'SP'
+
+    state_parsed = node.at_css('> state').text
+    expect(state_parsed).to eq 'SP'
+  end
+
+  it 'can be a country' do
+    address[:country] = 'BRA'
+
+    country_parsed = node.at_css('> country').text
+    expect(country_parsed).to eq 'BRA'
+  end
+end
+
+shared_examples 'when documents is assigned' do
+  it 'can be one document' do
```

Comment:
bro, split this specs into two.

One to check the `CPF` string and another to check the document number :)